windows/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -41,7 +41,7 @@ void kolibriScriptPath(char *buffer, const DWORD MAX_SIZE)
 		sprintf(err_message, "Error: the value of KOLIBRI_SCRIPT_DIR must be less than %d, but it was length %d. Please start Kolibri from the command line.", MAX_SIZE, bufsize);
 		window->sendTrayMessage("Kolibri", err_message);
 	}
-	char script_dirs[26] = "c:\\Python34\\Scripts";

```

Comment:
Can we find a way to remove the hard-coded `c:\` drive?  What if the user installed `Python` somewhere?

You can file a separate issue for this after testing, so we can close this.